inspire/modules/workflows/static/js/workflows/actions/core_approval.js

```diff
@@ -56,6 +57,8 @@ define(
         action_url: ""
       });
 
+      this.$pdf_submission_checkbox = $("[name='submission-data-pdf']");
```

Comment:
Sorry I forgot to mention before that I believe this selector could be part of the general component attribute `attr`.

This way it can look like:
```python
this.attributes({
        pdfCheckboxSelector: "[name='submission-data-pdf']",
        actionAcceptSelector: ".core-approval-action-accept",
```

and referenced like `$(this.attr.pdfCheckboxSelector)`